frontend/lib/pages/settings/settings_page.dart

```diff
@@ -8,8 +12,131 @@ class SettingsPage extends StatefulWidget {
 }
 
 class _SettingsPageState extends State<SettingsPage> {
+  @override
+  initState() {
+    super.initState();
+  }
+
+  bool enabled = false;
+
+  @override
+  Widget build(BuildContext context) {
+    Size deviceSize = MediaQuery.of(context).size;
+
+    return Scaffold(
+      appBar: buildAppBar(),
+      body: Stack(children: [
+        SettingsList(
+          sections: [
+            SpqSettingsSection(
+              title: const Padding(
+                padding: EdgeInsets.all(8.0),
+                child: Text(
+                  "@eric",
+                  style: TextStyle(
+                      color: spqBlack,
+                      fontSize: 22,
+                      fontWeight: FontWeight.bold),
+                ),
+              ),
+              tiles: [
+                buildSettingsTile("Account", "login"),
+                buildSettingsTile("Privacy and safety", "login"),
+                buildSettingsTile("Notifications", "login"),
+                buildSettingsTile("Content preferences", "login"),
+              ],
+            ),
+            SpqSettingsSection(
+              title: const Padding(
+                padding: EdgeInsets.all(8.0),
+                child: Text(
+                  "General",
+                  style: TextStyle(
+                      color: spqBlack,
+                      fontSize: 22,
+                      fontWeight: FontWeight.bold),
+                ),
+              ),
+              tiles: [
+                buildSettingsTile("Display and Sound", "login"),
+                buildSettingsTile("Data usage", "login"),
+                buildSettingsTile("Accessibility", "login"),
+                buildSettingsTile("About speaq", "login"),
+              ],
+            ),
+          ],
+        ),
+        Positioned(
+          child: Align(alignment: Alignment.bottomCenter,child: _buildLogoPicturres(deviceSize)), bottom: 20, height: deviceSize.height*0.1,
+        )
+      ]),
+    );
+  }
+
+  Widget _buildLogoPicturres(Size deviceSize) {
+    return SizedBox(width: deviceSize.width,
+      child: Image(height: deviceSize.height*0.05, width: deviceSize.width*0.3,image: const AssetImage('assets/logo.png')),
```

Comment:
We should probably use a SVG instead of a PNG so that loading this image from disk is fast on low end devices and scales well on big screens

Reply:
done